### PR TITLE
12.0 add stock_product_pack

### PR DESCRIPTION
**Pull request by ernestotejeda**
Supersedes https://github.com/OCA/stock-logistics-warehouse/pull/718

Depends on:

- [x] https://github.com/OCA/product-pack/pull/1

Cc @Tecnativa
TT17735

### CODE REVIEW

**Comment by ernestotejeda** [comment]
Ready to review

**Review by zeroheure — CHANGES_REQUESTED** [review]
stock_product_pack/readme/DESCRIPTION.rst
> This module allows you to get the right available *On Hand* and *Forecasted*
> quantities of the packs.

This module allows you to get *On Hand* and *Forecasted* values calculated from components of the pack.

File: stock_product_pack/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To configure this module, you need to:
+
+#. Go to *Inventory -> Master Data -> Product*, create a product and check
+   "Is Pack?" field, then set the products has to include in the Pack.
+#. Set type of product pack.
```
Comment (zeroheure) [inline stock_product_pack/readme/CONFIGURE.rst:5]: Wrong order
#. Go to *Inventory -> Master Data -> Product*, create a product and check
   *Is Pack?* field then set *Pack Type* and *Pack component price*.
#. Choose products to include in the pack.

File: stock_product_pack/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
```
Comment (zeroheure) [inline stock_product_pack/readme/CONFIGURE.rst:6]: This should be in USAGE.rst

File: stock_product_pack/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+This module allows you to get the right available *On Hand* and *Forecasted*
+quantities of the packs.
```
Comment (zeroheure) [inline stock_product_pack/readme/DESCRIPTION.rst:2]: Bad english 
_This module allows you to get On Hand and Forecasted values calculated from components of the pack._

**Review by pedrobaeza — APPROVED** [review]
Let's merge full pack of the pack modules :yum: for having the whole picture. We can bugfixing later with everything in place.

**Pull request merged**